epub33/rs/index.html

```diff
@@ -1852,33 +1867,73 @@ <h4>Scripting Considerations</h4>
 					</li>
 				</ul>
 
-				<p>This specification recommends the following practices for handling untrusted scripts:</p>
-
+				<p>
+					In handling untrusted scripts this specification recommends that Reading Systems establish 
```

Comment:
Hm. The way I read that section it does not offer alternatives but some additional considerations that RS should take into account. Which part do you feel explain alternatives?

cc @danielweck

Reply:
Ah, I think I know what you refer to: the first bullet item right after the quoted paragraph.

This is a leftover of the previous version that I did not remove. If you look at the discussion thread above, the main question is whether that paragraph is needed in the first place, ie, whether the normative requirement of how to handle origins needs some informative text for RSs that, for some reasons, cannot implement what is required.

Re-reading the whole set, even if that type of text _is_ needed, it may be a better option to add it to section 4.4 (a.k.a. line 802) as a (non normative) note. That would make it clearer that it is an advice for the RSs that cannot be fully conformant, but it would remove it from the security section where, I agree, it muddy the waters.

WDYT?

Reply:
I think it is strange to requires something with a MUST, but then acknowledge there are good reasons to not implement the MUST and provide alternative. If there exist good reasons why something should not be implemented, it seems the proper conformance statement is SHOULD, right? I think we should either require it, full stop, or strongly recommend it and give advice on what to do if you can't.

Reply:
+1 for requiring it, full stop.

Reply:
I still find this text confusing. Why would we only recommend unique origins to handle untrusted scripts but require them for all scripting?

Shouldn't this say something more like: "To limit the damage of untrusted scripts, this specification requires Reading Systems assign a unique origin ..."